### PR TITLE
Add birthday field & rework intercept validation

### LINKED ISSUE #100: Birthday field in profile
https://github.com/AreaFiftyLAN/lancie-api/issues/151

Add a birthday field for the user to verify he is lawfully permitted to drink.

### PR DESCRIPTION
**Pull request by martijnjanssen**
Fixes #100 
Path to the header was incorrect in the lancie-card element. Adds birthday field to the profile element in my-area and tickets-intercept. Tickets intercept profile validation is also redone to be more efficiënt.

### CODE REVIEW

File: src/lancie-my-area/my-area-profile.html [hunk 1]
```diff
@@ -171,8 +179,10 @@
             address: this.editingUser.profile.address,
             zipcode: this.editingUser.profile.zipcode,
             city: this.editingUser.profile.city,
-            gender: this.editingUser.profile.gender
+            gender: this.editingUser.profile.gender,
+            birthday: this.editingUser.profile.birthday
           };
+          console.log(this.$.ajaxUpdateProfile.body);
```
Comment (TimvdLippe) [inline src/lancie-my-area/my-area-profile.html]: Left-over debug statement

File: src/lancie-ticket-page/lancie-ticket-profile-intercept.html [hunk 2]
```diff
@@ -130,6 +88,7 @@
 
       onSubmit: function() {
         if (this.allFilledIn()) {
+          this.complete = true;
```
Comment (TimvdLippe) [inline src/lancie-ticket-page/lancie-ticket-profile-intercept.html]: Is this really necessary?

File: src/lancie-my-area/profile-edit-form.html [hunk 3]
```diff
@@ -10,36 +10,44 @@
...
+      </paper-item-body>
+    </paper-icon-item>
+    <paper-icon-item>
       <iron-icon icon="communication:phone" item-icon></iron-icon>
       <paper-item-body>
         <paper-input label="Phone Number" type="text" value="{{profile.phoneNumber}}"></paper-input>
       </paper-item-body>
     </paper-icon-item>
-    <paper-icon-item tabindex="-1">
+    <paper-icon-item>
```
Comment (TimvdLippe) [inline src/lancie-my-area/profile-edit-form.html]: We have to make sure tabbing still hits all input fields

File: src/lancie-ticket-page/lancie-ticket-profile-intercept.html (hunk 2 above)
Comment (martijnjanssen) [inline src/lancie-ticket-page/lancie-ticket-profile-intercept.html]: Yes, it is. The complete variable shows an error message to the user if the complete variable of false. The message lets the user know that some fields are still not filled in.

**Comment by martijnjanssen** [comment]
@TimvdLippe Could you look at this PR, it has been open for quite some time and is ready to merge in my opinion.

**Comment by TimvdLippe** [comment]
I cant register a new user due to a 403 on the server. This has to be fixed and then I can test this PR

**Comment by martijnjanssen** [comment]
I'm quite sure the issue was a 415 on the server on the register request because we did not send the `application/json` header. This should be resolved now.

**Comment by TimvdLippe** [comment]
Apparently master of the API is broken as CSRF tokens are expected by the API even though they should be disabled

**Pull request merged**